products.php

```diff
@@ -0,0 +1,51 @@
+<?php
+require_once 'src/Models/CategoryModel.php';
+require_once 'src/Services/ProductsDisplayService.php';
+require_once 'src/Models/ProductModel.php';
+require_once 'src/Factory/furnitureDatabaseConnector.php';
+require_once 'src/Entities/ProductsEntity.php';
+
+$db = furnitureDatabaseConnector::connect();
+define('NUM_OF_CATEGORIES', 11);
+$id = $_GET["id"];
```

Comment:
This crashes if there is no id in the `$_GET`.

![image](https://github.com/iO-Academy/2024-mar-sde-furniture-store-panda-monium/assets/24191795/b5b1885c-c45c-42ed-a0ac-66a498af65c3)